### ISSUE TITLE
running lsquic locally 

### ISSUE THREAD

**Issue by yc1997427** [litespeedtech/lsquic#153]
Hi, may I ask which lsquic examples would be better for testing locally? Like binding server with local server ip address and port, client request that server. I saw there was http_server, echo_server seemed do not support binding with ip address and port.

**Comment by dtikhonov** [comment]
Sure, you can bind to a particular IP address, you can even bind to a particular interface:

```
  - s 192.168.0.1:12345
```

or

```
  -s 192.168.0.1:12345,eth0    # You'll need privileges for that
```

Note that all example programs, servers and clients, support the `-s` flag.  You can pass more than one `-s` flag to servers.

**Comment by yc1997427** [comment]
Thanks. I just tried, but got error:
./http_server -s 192.168.0.1:4434 -p / -o version=Q050
20:40:38.647 [NOTICE] Document root is not set: start in Interop Mode
20:40:38.647 [WARN] Not a single service specified.  Use -c option.
20:40:38.648 [WARN] bind failed: Cannot assign requested address
20:40:38.648 [ERROR] could not prep

Cannot assign requested address, which is exactly the same error as I tested quiche. Probably that is our server's own problem, still trying to solve it.
